components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -219,8 +245,9 @@ public function options( $options, $pod ) {
 
 				}
 
-				$options['pods-pfat']['pfat_archive']['data'] = array( null => __( 'No Archive view template', 'pods' ) ) + ( array_combine( $this->get_template_titles(), $this->get_template_titles() ) );
-				$options['pods-pfat']['pfat_single']['data']  = array_combine( $this->get_template_titles(), $this->get_template_titles() );
+				$template_titles = array_combine( $this->get_template_titles(), $this->get_template_titles() );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 30 spaces but found 1 space

<sub>hash: aba4108c3bb8d95f4b1d86c0975b69e3</sub>

Reply:
```suggestion
				$template_titles = array_combine( $this->get_template_titles(), $this->get_template_titles() );

```